releases/codefresh.yaml

```diff
@@ -100,6 +105,12 @@ releases:
       serviceType: NodePort
 {{- end }}
 
+    nomios:
+      service:
+{{- if eq (env "CODEFRESH_INGRESS_CLASS" | default "nginx") "alb" }}
+        type: NodePort
+{{- end }}
+
```

Comment:
The whole block should be inside the conditional
```suggestion
{{- if eq (env "CODEFRESH_INGRESS_CLASS" | default "nginx") "alb" }}
    nomios:
      service:
        type: NodePort
{{- end }}

```

Reply:
You need to delete the section above

Reply:
Done